examples/ccsds-frames/src/main/yamcs/etc/yamcs.ccsds-frames.yaml

```diff
@@ -86,37 +86,37 @@ dataLinks:
     frameHeaderErrorControlPresent: true #2 bytes of reed-solomon (10,6) in the header of all frames
     insertZoneLength: 0
     errorDetection: CRC16 #NONE, CRC16 or CRC32 (only for USLP)
-    clcwStream: "clcw"    #publish the received CLCW (what they call OCF in the CCSDS AOS/TM/USLP standards) to this stream, to be used by the FOP1
+    clcwStream: "clcw" #publish the received CLCW (what they call OCF in the CCSDS AOS/TM/USLP standards) to this stream, to be used by the FOP1
     goodFrameStream: "good_frame_stream" #publish the good frames on this stream
-    badFrameStream: "bad_frame_stream"   #publish the bad frames on this stream
+    badFrameStream: "bad_frame_stream" #publish the bad frames on this stream
     virtualChannels:
-          - vcId: 0
```

Comment:
same as above

Reply:
As agreed, we keep the formatting changes for consistency